experience.tex

```diff
@@ -6,33 +6,34 @@ \section{EXPERIENCE}
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Worked with the Supply Chain team to determine requirements and metric thresholds for displaying data to
-        the service providers based on their performance for previously completed work items.
+        Assisted building and maintaing greenfield iOS application written in SwiftUI to replace legacy Xamarin application,
+        reducing crash rate from 9\% to < 1\%, reducing time for local development and time to ship a new version of the app,
+        as well as added thorough logging to the application to make errors actionable for development team.
     }
     \item{
-        Utilized AWS Elasticsearch to improve filtering for assets by properties and to quickly return large lists of
-        matching data to the primary asset web service, lowering asset search response times for the end user.
+        Containerized legacy C\# applications to run in serverless AWS Fargate, moving secrets to HashiCorp Vault to be fetched
```

Comment:
Explain business impact here - why make the move or containerize?

Reply:
Fair - here's what comes to mind:
- Modernized deployment strategy
- Modernized API secret storage
- Move API off of shared web farm with other APIs (& moved to serverless) so it can scale to the moon 🚀


I need to think about how to condense that into a few words that doesn't balloon the size of this bullet point

Reply:
Added some impact (deployment + scalability) to this bullet in c779aa8fd047153d7179eedc881201d497159268